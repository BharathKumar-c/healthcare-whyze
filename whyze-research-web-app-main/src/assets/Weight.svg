<svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg">
<rect width="40" height="40" rx="20" fill="#F7FCD9"/>
<g clip-path="url(#clip0_965_65567)">
<path d="M27.9877 8.48828H24.8002H15.2002H12.0127C10.0627 8.48828 8.4502 10.1008 8.4502 12.0508V27.9883C8.4502 29.9383 10.0627 31.5508 12.0127 31.5508H27.9502C29.9002 31.5508 31.5127 29.9383 31.5127 27.9883V12.0133C31.5502 10.0633 29.9377 8.48828 27.9877 8.48828ZM24.1252 9.80078V12.2758C24.1252 13.2133 23.3377 14.0008 22.4002 14.0008H20.5627L21.4627 12.3883C21.6502 12.0883 21.5377 11.6758 21.2002 11.4883C20.9002 11.3008 20.4877 11.4133 20.3002 11.7508L19.0252 14.0008H17.5627C16.6252 14.0008 15.8752 13.2133 15.8752 12.2758V9.80078H24.1252ZM30.2377 27.9883C30.2377 29.2258 29.2252 30.2383 27.9877 30.2383H12.0127C10.7752 30.2383 9.7627 29.2258 9.7627 27.9883V12.0133C9.7627 10.7758 10.7752 9.76328 12.0127 9.76328H14.5252V12.2383C14.5252 13.9258 15.8752 15.2758 17.5627 15.2758H19.4002H22.3627C24.0502 15.2758 25.4002 13.9258 25.4002 12.2383V9.80078H27.9127C29.1502 9.80078 30.1627 10.8133 30.1627 12.0508V27.9883H30.2377Z" fill="#718E0B"/>
</g>
<defs>
<clipPath id="clip0_965_65567">
<rect width="24" height="24" fill="white" transform="translate(8 8)"/>
</clipPath>
</defs>
</svg>
