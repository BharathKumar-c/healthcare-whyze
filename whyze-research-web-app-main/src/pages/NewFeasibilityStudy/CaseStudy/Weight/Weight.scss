.weight-container {

   .weight-wrapper {
      display: flex;
      flex-direction: row;
      justify-content: flex-start;

      &_dragicon {
         margin-top: 1.5rem;
         margin-right: 1rem;
      }

      &_min-inputbox {
         width: 130px;

         div {
            span {
               display: flex;
            }
         }
      }

      &_max-inputbox {
         width: 150px;
         margin-top: 1.7rem !important;

         div {
            span {
               display: flex;
            }
         }
      }

      &_weight-unit-select {
         width: 150px;
         margin-top: 1.7rem;
      }

      @media screen and (max-width: 816px) {
         &_min-inputbox {
            width: 180px;
         }

         &_max-inputbox {
            width: 180px;
            margin-top: 1.7rem !important;
         }

         &_weight-unit-select {
            width: 150px;
            margin-top: 1.7rem;
         }
      }
   }

}