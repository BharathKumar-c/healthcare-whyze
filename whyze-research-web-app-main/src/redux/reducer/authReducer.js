import { createSlice } from '@reduxjs/toolkit';
import jwt_decode from 'jwt-decode';

const initialState = {
  isLoggedIn: loginCheck(),
  accessToken: null,
  refreshToken: null,
  isLoading: false,
  isDisableForgot: false,
  message: '',
  userDetails: {},
  profileCount: 0,
};

function loginCheck() {
  const credentials = JSON.parse(localStorage.getItem('credentials'));
  if (credentials?.access_token) {
    const { exp } = jwt_decode(credentials.access_token);
    if (Date.now() <= exp * 1000) {
      return true;
    }
  }
  return false;
}

const authSlice = createSlice({
  name: 'authAction',
  initialState: initialState,
  reducers: {
    setLoginStatus: (state, { payload }) => {
      state.isLoggedIn = payload;
    },
    setAccessToken: (state, { payload }) => {
      state.accessToken = payload;
    },
    setRefreshToken: (state, { payload }) => {
      state.refreshToken = payload;
    },
    setLoading: (state, { payload }) => {
      state.isLoading = payload;
    },
    setDisableForgot: (state, { payload }) => {
      state.isDisableForgot = payload;
    },
    setResetMessage: (state, { payload }) => {
      state.message = payload;
    },
    setUserDetails: (state, { payload }) => {
      state.userDetails = payload;
    },
    setProfileCount: (state) => {
      state.profileCount = state.profileCount + 1;
    },
  },
});

export const {
  setLoginStatus,
  setAccessToken,
  setRefreshToken,
  setLoading,
  setDisableForgot,
  setResetMessage,
  setUserDetails,
  setProfileCount,
} = authSlice.actions;

export default authSlice.reducer;
