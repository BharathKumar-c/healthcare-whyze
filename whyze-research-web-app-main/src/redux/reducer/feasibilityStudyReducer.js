import { createSlice } from '@reduxjs/toolkit';
import { v4 as uuid } from 'uuid';

export const initialState = {
  segments: [
    {
      name: 'Segment-1',
      isDone: false,
      activeTab: {
        name: 'feasibility',
        index: 0,
      },
      tabs: [
        {
          name: 'feasibility',
          title: 'Feasibility',
          isActive: true,
          position: 'left',
        },
      ],
      feasibilityStudy: [
        {
          name: 'Indication',
          case: 'Indication',
          condition: 'Inclusion',
          caseName: '',
          child: [],
        },
        {
          name: 'Demographics',
          case: 'Demographics',
          condition: 'Inclusion',
          caseName: '',
          child: [
            {
              id: uuid().slice(0, 8),
              condition: 'Gender',
              field: 'gender',
              gender: '',
            },
            {
              id: uuid().slice(0, 8),
              condition: 'Age range',
              field: 'age',
              ageRange: {
                min: 0,
                max: 0,
              },
            },
          ],
        },
      ],
      country: [],
      location: [],
      selectedCountries: [],
      totalConnectedNotConnected: {},
      sites: [],
      sitesContacted: [],
      sitesInitiated: [],
      preferredSites: [],
      patientCount: {
        feasibility: 0,
        location: 0,
      },
      siteCount: {
        feasibility: 0,
        location: 0,
      },
    },
  ],
  cardValue: {
    case: 'Indication',
    condition: '',
    index: 0,
  },
  activeSegment: {
    name: '',
    index: 0,
  },
};

const feasibilityStudySlice = createSlice({
  name: 'feasibilityStudyAction',
  initialState: initialState,
  reducers: {
    setCardDetails: (state, { payload }) => {
      state.cardValue = payload;
    },
    setSegments: (state, { payload }) => {
      state.segments = payload;
    },
    setActiveSegment: (state, { payload }) => {
      state.activeSegment = payload;
    },
    resetSegments: (state) => {
      state.segments = initialState.segments;
      state.cardValue = {
        case: 'Indication',
        condition: '',
        index: 0,
      };
      state.activeSegment = {
        name: '',
        index: 0,
      };
    },
  },
});

export const { setCardDetails, setSegments, setActiveSegment, resetSegments } =
  feasibilityStudySlice.actions;

export default feasibilityStudySlice.reducer;
