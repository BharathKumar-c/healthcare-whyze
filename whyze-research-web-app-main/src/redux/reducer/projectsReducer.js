import { createSlice } from '@reduxjs/toolkit';

const initialState = {
  projects: [],
  projectId: '',
  clinicalTrial: '',
  sponsor: '',
  description: '',
  measurement: 'Metric',
  standards: '',
  segmentId: [],
};

const projectSlice = createSlice({
  name: 'projectAction',
  initialState: initialState,
  reducers: {
    setProjects: (state, { payload }) => {
      state.projects = payload;
    },
    setProjectId: (state, { payload }) => {
      state.projectId = payload;
    },
    setClinicalTrial: (state, { payload }) => {
      state.clinicalTrial = payload;
    },
    setSponsor: (state, { payload }) => {
      state.sponsor = payload;
    },
    setDescription: (state, { payload }) => {
      state.description = payload;
    },
    setStandards: (state, { payload }) => {
      state.standards = payload;
    },
    setMeasurement: (state, { payload }) => {
      state.measurement = payload;
    },
    setSegmentId: (state, { payload }) => {
      state.segmentId = payload;
    },
    setProjectSettings: (state, { payload }) => {
      state.projectId = payload.projectId;
      state.segmentId = payload.segmentId;
      state.clinicalTrial = payload.clinicalTrial;
      state.sponsor = payload.sponsor;
      state.description = payload.description;
      state.measurement = payload.measurement;
      state.standards = payload.standards;
    },
    clearSettings: (state) => {
      state.clinicalTrial = '';
      state.sponsor = '';
      state.description = '';
      state.measurement = 'Metric';
      state.standards = '';
      state.projectId = '';
      state.segmentId = [];
    },
  },
});

export const {
  setProjects,
  setProjectId,
  setProjectSettings,
  setClinicalTrial,
  setDescription,
  setSponsor,
  setMeasurement,
  setStandards,
  setSegmentId,
  clearSettings,
} = projectSlice.actions;

export default projectSlice.reducer;
