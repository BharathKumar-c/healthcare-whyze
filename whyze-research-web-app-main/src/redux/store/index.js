import { configureStore } from '@reduxjs/toolkit';
import reducer from '../reducer/combinedReducer';

const store = configureStore({
  reducer: reducer,

  middleware: (getDefaultMiddleware) =>
    getDefaultMiddleware({
      serializableCheck: false,
    }),
});

export default store;
