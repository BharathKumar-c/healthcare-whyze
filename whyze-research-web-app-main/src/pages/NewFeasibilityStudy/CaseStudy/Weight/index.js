import React, { useState } from 'react';
import { Input, Select } from '../../../../components/basic';
import PropTypes from 'prop-types';
import { ConditionContainer } from '../../../../components/shared';
import { Row, Tag } from 'antd';
import { Drag, Vector } from '../../../../assets';
import './Weight.scss';
import { useDispatch } from 'react-redux';
import useNewFeasibilityStudyService from '../../useNewFeasibilityStudyService';
import { useSelector } from 'react-redux';
import { setSegments } from '../../../../redux/reducer/feasibilityStudyReducer';
import { weightConst } from '../../../../constant/ConstantTexts';
import { weightRange } from '../../../../constant';
import { totalPatientFormatter } from '../../../../utils/calculations';

const Weight = (props) => {
  const [isError, setIsError] = useState(false);
  const { segments, activeSegment } = useSelector(
    (state) => state.feasibilityStudyReducer,
  );

  const dispatch = useDispatch();

  const { optimusValue, isDisableInput } = useNewFeasibilityStudyService();

  const handleInputChange = ({ e, mainIndex, subIndex }) => {
    const study = JSON.parse(JSON.stringify(segments));

    study[activeSegment.index].feasibilityStudy[mainIndex].child[
      subIndex
    ].weightRange = {
      ...study[activeSegment.index]?.feasibilityStudy[mainIndex]?.child[
        subIndex
      ]?.weightRange,
      [e.target.name]: Number(e.target.value),
    };
    setIsError(e.target.value === '');
    dispatch(setSegments(study));
    if (
      study[activeSegment.index].feasibilityStudy[props.parentIndex].child[0]
        .weightRange.range.length > 0
    ) {
      optimusValue({
        id: study[activeSegment.index].feasibilityStudy[mainIndex].child[
          subIndex
        ]?.id,
        data: study[activeSegment.index]?.feasibilityStudy,
      });
    }
  };

  const handleSelect = (value) => {
    const study = JSON.parse(JSON.stringify(segments));
    study[activeSegment.index].feasibilityStudy[
      props.parentIndex
    ].child[0].weightRange = {
      ...study[activeSegment.index]?.feasibilityStudy[props.parentIndex]
        ?.child[0]?.weightRange,
      range: value,
    };
    dispatch(setSegments(study));
    optimusValue({
      id: props.caseDetails[0].id,
      data: study[activeSegment.index]?.feasibilityStudy,
    });
  };

  return (
    <div className="weight-container">
      {props.caseDetails.map((c, index) => (
        <ConditionContainer
          className={'condition-container'}
          isActive
          isVertical
          key={index}
        >
          <div className="weight-wrapper">
            <div className="weight-wrapper_dragicon">
              <Drag />
            </div>
            <div className="weight-wrapper_min-inputbox">
              <Input
                lable={weightConst.weightRange}
                type={weightConst.weightType}
                width={100}
                name={weightConst.weightMin}
                value={c?.weightRange?.min === 0 ? '' : c?.weightRange?.min}
                placeholder={weightConst.weightFrom}
                handleInputChange={(e) =>
                  handleInputChange({
                    e,
                    mainIndex: props.parentIndex,
                    subIndex: index,
                    id: c.weightRange.id,
                  })
                }
                disabled={isDisableInput}
                isError={isError}
                errorText={weightConst.weightminErrorText}
                onWheel={(e) => e.target.blur()}
              />
            </div>
            <div className="weight-wrapper_max-inputbox">
              <Input
                type={weightConst.weightType}
                width={100}
                name={weightConst.weightMax}
                value={c?.weightRange?.max === 0 ? '' : c?.weightRange?.max}
                placeholder={weightConst.weightTo}
                handleInputChange={(e) =>
                  handleInputChange({
                    e,
                    mainIndex: props.parentIndex,
                    subIndex: index,
                  })
                }
                disabled={isDisableInput}
                isError={isError}
                errorText={weightConst.weightmaxErrorText}
                onWheel={(e) => e.target.blur()}
              />
            </div>
            <div
              className="weight-wrapper_weight-unit-select"
              id="weight-wrapper_weight-unit-select"
            >
              <Select
                ariaLabel={'weight-unit'}
                options={weightRange}
                handleSelect={handleSelect}
                getPopupContainer={() =>
                  document.getElementById('weight-wrapper_weight-unit-select')
                }
                value={props?.caseDetails[0].weightRange.range}
              />
            </div>
          </div>

          <Row className="indication-no-patient-tag-row">
            <Tag className="indication-no-patient-tag">
              <Vector />
              {totalPatientFormatter(c.count || 0)}
            </Tag>
          </Row>
        </ConditionContainer>
      ))}
    </div>
  );
};

Weight.propTypes = {
  caseDetails: PropTypes.array,
  parentIndex: PropTypes.number,
  isChild: PropTypes.bool,
  handleAddCondition: PropTypes.func,
  handleChangeCaseName: PropTypes.func,
};

Weight.defaultProps = {
  caseDetails: [],
  parentIndex: 0,
  isChild: false,
  handleAddCondition: () => {},
  handleChangeCaseName: () => {},
};
export default Weight;
