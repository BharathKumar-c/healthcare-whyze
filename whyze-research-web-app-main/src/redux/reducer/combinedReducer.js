import { combineReducers } from 'redux';
import feasibilityStudy from './feasibilityStudyReducer';
import project from './projectsReducer';
import auth from './authReducer';

export default combineReducers({
  feasibilityStudyReducer: feasibilityStudy,
  authReducer: auth,
  projectReducer: project,
});
