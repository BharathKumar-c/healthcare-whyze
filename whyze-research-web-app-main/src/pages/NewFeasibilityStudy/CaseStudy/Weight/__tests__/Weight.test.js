import React from 'react';
import {
  act,
  fireEvent,
  render,
  screen,
  waitFor,
} from '@testing-library/react';
import { Provider } from 'react-redux';

import store from '../../../../../redux/store';
import Weight from '..';
import { setSegments } from '../../../../../redux/reducer/feasibilityStudyReducer';

function renderWithContext(element) {
  const { container, getByText } = render(
    <Provider store={store}>{element}</Provider>,
  );
  return { container, store, getByText };
}

const caseDetails = {
  name: 'Weight',
  case: 'Weight',
  condition: 'Inclusion',
  caseName: '',
  child: [
    {
      id: '1892f325',
      condition: 'Weight range',
      field: 'weight',
      weightRange: {
        min: 10,
        max: 30,
        range: '',
      },
    },
  ],
};

describe('create testcases for Weight', () => {
  test('should create snapshots for weight', () => {
    const { container } = renderWithContext(
      <Weight caseDetails={caseDetails.child} />,
    );

    act(() => {
      const { segments, activeSegment } = JSON.parse(
        JSON.stringify(store.getState().feasibilityStudyReducer),
      );
      segments[activeSegment.index].feasibilityStudy.push(caseDetails);
      store.dispatch(setSegments(segments));
    });

    expect(container).toMatchSnapshot();
  });

  test('Should render the drag icon correctly', () => {
    const { container } = renderWithContext(
      <Weight caseDetails={caseDetails.child} />,
    );
    const dragIcon = container.getElementsByClassName(
      'weight_range-wrapper_drag-icon',
    );

    expect(dragIcon[0].textContent).toBe('Drag.svg');
  });

  test('Should render min input field correctly', async () => {
    const { container } = renderWithContext(
      <Weight caseDetails={caseDetails.child} parentIndex={2} />,
    );
    const minInputField = container.querySelector('input[name="min"');
    expect(minInputField).toBeInTheDocument();
    expect(minInputField.value).toBe('10');

    fireEvent.change(minInputField, { target: { value: 30 } });

    await waitFor(() => {
      const { segments, activeSegment } =
        store.getState().feasibilityStudyReducer;

      expect(
        segments[activeSegment.index]?.feasibilityStudy[2].child[0].weightRange
          .min,
      ).toEqual('30');
    });
  });

  test('Should render max input field correctly', async () => {
    const { container } = renderWithContext(
      <Weight caseDetails={caseDetails.child} parentIndex={2} />,
    );
    const maxInputField = container.querySelector('input[name="max"');
    expect(maxInputField).toBeInTheDocument();
    expect(maxInputField.value).toBe('30');

    fireEvent.change(maxInputField, { target: { value: 60 } });

    await waitFor(() => {
      const { segments, activeSegment } =
        store.getState().feasibilityStudyReducer;

      expect(
        segments[activeSegment.index]?.feasibilityStudy[2].child[0].weightRange
          .max,
      ).toEqual('60');
    });
  });

  test('Should render unit select field correctly', async () => {
    const { container } = renderWithContext(
      <Weight caseDetails={caseDetails.child} parentIndex={2} />,
    );
    const unitSelectField = screen.getByRole('combobox');
    expect(unitSelectField).toBeInTheDocument();
    expect(unitSelectField).toHaveAttribute('aria-label', 'weight-unit');
    const defaultValue = container.getElementsByClassName(
      'ant-select-selection-item',
    );
    expect(defaultValue[0].textContent).toBe('Kg');

    fireEvent.change(unitSelectField, { target: { value: 'lbs' } });
    const option = screen.getByTitle('Lbs');
    fireEvent.click(option);

    await waitFor(() => {
      const { segments, activeSegment } =
        store.getState().feasibilityStudyReducer;

      expect(
        segments[activeSegment.index]?.feasibilityStudy[2].child[0].weightRange
          .range,
      ).toEqual('lbs');
    });
  });
});
