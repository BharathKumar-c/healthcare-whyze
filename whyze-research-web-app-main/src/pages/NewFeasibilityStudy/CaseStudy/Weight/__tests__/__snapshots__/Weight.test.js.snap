// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create testcases for Weight should create snapshots for weight 1`] = `
<div>
  <div
    class="weight_container"
  >
    <div
      class="ant-space ant-space-horizontal ant-space-align-center condition-container"
      style="border: 2px solid #d3d6d9; display: flex; flex-direction: column; margin-bottom: 0.5rem; padding: 20px 30px 12px 20px;"
    >
      <div
        class="ant-space-item"
      >
        <div
          class="ant-row weight_range-wrapper"
          style="margin-left: -16px; margin-right: -16px;"
        >
          <div
            class="ant-col weight_range-wrapper_drag-icon ant-col-sm-1 ant-col-md-2 ant-col-lg-1 ant-col-xl-1"
            style="padding-left: 16px; padding-right: 16px;"
          >
            <svg>
              Drag.svg
            </svg>
          </div>
          <div
            class="ant-col"
            style="padding-left: 16px; padding-right: 16px;"
          >
            <div
              class="ant-row"
              style="margin-left: -16px; margin-right: -16px;"
            >
              <div
                class="ant-col weight_range-wrapper_Input-box ant-col-sm-8 ant-col-md-8 ant-col-lg-8 ant-col-xl-8 ant-col-xxl-8"
                style="padding-left: 16px; padding-right: 16px;"
              >
                <div>
                  <span
                    class="ant-typography input-lable"
                  >
                    Weight Range
                  </span>
                  <input
                    class="ant-input"
                    name="min"
                    placeholder=""
                    style="border-radius: 0px; width: 100px;"
                    type="number"
                    value="10"
                  />
                </div>
              </div>
              <div
                class="ant-col weight_range-wrapper_Input-box1 ant-col-sm-8 ant-col-md-8 ant-col-lg-8 ant-col-xl-8 ant-col-xxl-8"
                style="padding-left: 16px; padding-right: 16px;"
              >
                <div>
                  <input
                    class="ant-input"
                    name="max"
                    placeholder=""
                    style="border-radius: 0px; width: 100px;"
                    type="number"
                    value="30"
                  />
                </div>
              </div>
              <div
                class="ant-col ant-col-md-8"
                style="padding-left: 16px; padding-right: 16px;"
              >
                <div
                  class="alignment"
                >
                  <div
                    aria-label="weight-unit"
                    class="ant-select weight_range-wrapper_Input-box1 ant-select-single ant-select-show-arrow"
                  >
                    <div
                      class="ant-select-selector"
                    >
                      <span
                        class="ant-select-selection-search"
                      >
                        <input
                          aria-activedescendant="rc_select_TEST_OR_SSR_list_0"
                          aria-autocomplete="list"
                          aria-controls="rc_select_TEST_OR_SSR_list"
                          aria-haspopup="listbox"
                          aria-label="weight-unit"
                          aria-owns="rc_select_TEST_OR_SSR_list"
                          autocomplete="off"
                          class="ant-select-selection-search-input"
                          id="rc_select_TEST_OR_SSR"
                          readonly=""
                          role="combobox"
                          style="opacity: 0;"
                          type="search"
                          unselectable="on"
                          value=""
                        />
                      </span>
                      <span
                        class="ant-select-selection-item"
                        title="Kg"
                      >
                        Kg
                      </span>
                    </div>
                    <span
                      aria-hidden="true"
                      class="ant-select-arrow"
                      style="user-select: none;"
                      unselectable="on"
                    >
                      <span
                        aria-label="down"
                        class="anticon anticon-down ant-select-suffix"
                        role="img"
                      >
                        <svg
                          aria-hidden="true"
                          data-icon="down"
                          fill="currentColor"
                          focusable="false"
                          height="1em"
                          viewBox="64 64 896 896"
                          width="1em"
                        >
                          <path
                            d="M884 256h-75c-5.1 0-9.9 2.5-12.9 6.6L512 654.2 227.9 262.6c-3-4.1-7.8-6.6-12.9-6.6h-75c-6.5 0-10.3 7.4-6.5 12.7l352.6 486.1c12.8 17.6 39 17.6 51.7 0l352.6-486.1c3.9-5.3.1-12.7-6.4-12.7z"
                          />
                        </svg>
                      </span>
                    </span>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
